package.json

```diff
@@ -23,7 +23,7 @@
     "broccoli-merge-trees": "^1.0.0",
     "ember-cli-babel": "^6.6.0",
     "es5-shim": "~4.5.2",
-    "pixi.js": "~3.0.9"
+    "pixi.js": "~4.5.2"
```

Comment:
Not familiar with the details of `peerDependencies` actually but probably good ;)